### ISSUE TITLE
pip doesn't install requests dependency

### ISSUE THREAD

**Issue by foxy82** [dcnielsen90/python-bravia-tv#9]
```
(bravia) obs@computer:/mnt/c/Users/obs/git/scripts$ pip install bravia-tv
Requirement already satisfied: bravia-tv in /mnt/c/Users/obs/pyvenv/bravia/lib/python3.8/site-packages (1.0.3)
(bravia) obs@computer:/mnt/c/Users/obs/git/scripts$ python tv.py
Traceback (most recent call last):
  File "tv.py", line 1, in <module>
    from bravia_tv import BraviaRC
  File "/mnt/c/Users/obs/pyvenv/bravia/lib/python3.8/site-packages/bravia_tv/__init__.py", line 7, in <module>
    from .braviarc import BraviaRC
  File "/mnt/c/Users/obs/pyvenv/bravia/lib/python3.8/site-packages/bravia_tv/braviarc.py", line 15, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
(bravia) obs@computer:/mnt/c/Users/obs/git/scripts$
```

**Comment by foxy82** [comment]
Obviously works fine after: `pip install requests`

**Comment by dcnielsen90** [comment]
Good catch -- It'll be fixed next release.

**Comment by dcnielsen90** [comment]
This is now fixed as of 1.0.4.
